### ISSUE TITLE
Why is rafce not working in jsx files

### ISSUE THREAD

**Issue by M23jJohn** [bajrangCoder/acode-react-snippets#12]

**Comment by bajrangCoder** [comment]
Is other snippets works ?

**Comment by M23jJohn** [comment]
yes but in jsx that rafce does not work it will only work in tsx files

**Comment by bajrangCoder** [comment]
Share a recording

**Comment by M23jJohn** [comment]
https://github.com/user-attachments/assets/ec07603c-87b0-430f-b1b8-1258c14e6f22

SLR

**Comment by bajrangCoder** [comment]
It seems that another plugin might be overriding it. Do you have any other snippets plugin installed for React on your Acode?

**Comment by M23jJohn** [comment]
i only use snippets which only works on css and HTML and the React Native snippets too

**Comment by bajrangCoder** [comment]
Try by disabling React Native snippets

**Comment by M23jJohn** [comment]
Still nothing, can't upload the file for the recoding here can it be by email instead

**Comment by bajrangCoder** [comment]
You can upload here , check there is image icon in the input box 

Btw tell me, what's the syntax highlighting selected? For jsx from pencil icon

**Comment by bytebrain3** [comment]
I find the fix .
You need to change the syntax highlighting to tsx . Then it's wok perfectly.

**Comment by bajrangCoder** [comment]
> I find the fix . You need to change the syntax highlighting to tsx . Then it's wok perfectly.

but that's a work around , not a solution

let me check , as it should work in both `jsx` and `tsx`

**Comment by bajrangCoder** [comment]
> I find the fix . You need to change the syntax highlighting to tsx . Then it's wok perfectly.

It should work 
https://github.com/bajrangCoder/acode-react-snippets/blob/5c4fc524409a98ba11a2ea177ca04210b76c32f7/src/snippets.ts#L19

Share some context about your environment, as it's working fine for me in both jsx,tsx mode
